### ISSUE TITLE
key not found

### ISSUE THREAD

**Issue by fommil** [scalameta/scalafmt#1203]
I've tried 1.5.1 and 1.6.0-RC1 and get the same problem. Screenshot attached of the exception that causes this to appear in sbt

```
[error] Error in /home/fommil/Work/.../Recorder.scala: java.util.NoSuchElementException: key not found: -5260204305981373584
```

for the source code

```scala
package foo

object Recorder {
  sealed abstract class Ast[A]
  final case class Record()
}
```

and configuration

```
verticalMultilineAtDefinitionSite = true
```

![2018-05-30-115201_1918x1078_scrot](https://user-images.githubusercontent.com/1914041/40716174-f91df99c-63ff-11e8-97cf-5da3b6b297a1.png)

**Comment by fommil** [comment]
@pjrt

**Comment by pjrt** [comment]
This doesn't happen with `verticalMultilineAtDefinitionSite = false`? That error normally happens when the internal map Scalafmt uses doesn't contain a given token, but not sure how `verticalMultilineAtDefinitionSite` would affect that. 

Any ideas, @olafurpg ?

**Comment by olafurpg** [comment]
I am able to reproduce, thanks @fommil for the minimal example.

Pending fix in #1213
